core/info/release-notes/0.105.txt

```diff
@@ -1 +1,6 @@
-Nothing for now.
+1. Org
+
+Spacemacs now uses the `org` version from the org ELPA repository instead of the
+one shipped with emacs. Then, any `org` related code should not be loaded before
+`dotspacemacs/user-config`, otherwise both versions will be loaded and will
+conflict.
```

Comment:
Actually this change has been made in 0.104 ;-) I guess we should document it in the FAQ instead.